app/models/order_detail.rb

```diff
@@ -771,6 +747,34 @@ def problem_order?
     problem
   end
 
+  def can_dispute?
+    in_review? && dispute_at.blank?
+  end
```

Comment:
Not quite following the question. This needs to be a little more restrictive than `in_review? && !in_dispute?`. An order is no longer `in_dispute?` once `dispute_resolved` is set or the order is canceled, but it does need to be `!can_dispute?`.